MQTicketService/startApp.sh

```diff
@@ -20,4 +20,4 @@ sleep 30
 # Set environment
 . /opt/mqm/bin/setmqenv -s
 # Execute TicketGenerator
-/opt/mqm/java/jre64/jre/bin/java -cp ./jarFiles/*: -Djava.library.path=/opt/mqm/java/lib64 com.ibm.mq.badge.Manager
+/jdk-11.0.13+8-jre/bin/java -cp /data/TicketGenerator/target/TicketGenerator-1.4.jar: -Djava.library.path=/opt/mqm/java/lib64 com.ibm.mq.badge.Manager
```

Comment:
I think that 

` -Djava.library.path=/opt/mqm/java/lib64`

was needed because the app was running in bindings mode. Now that it is running in client mode this shouldn't be needed. You should test with 

`/jdk-11.0.13+8-jre/bin/java -cp /data/TicketGenerator/target/TicketGenerator-1.4.jar com.ibm.mq.badge.Manager`

Reply:
Updated as required.